infra/taskcluster-hook-pipeline-start.json

```diff
@@ -29,25 +29,27 @@
             "cache": {},
             "capabilities": {},
             "command": [
-                "/hello"
+                "/bin/bash",
+                "-lcx",
+                "git clone https://github.com/mozilla/bugbug && cd bugbug && git checkout data-pipeline-definition && pip install taskcluster json-e requests pyyaml && python infra/spawn_data_pipeline.py data-pipeline.yml"
```

Comment:
This should be an image, instead of doing the git clone + dependency installation at runtime.

We can do it as a follow-up (just remember to file an issue if you want to do it in a follow-up)

Reply:
I think @La0 suggested that this should be a task-boot commando I guess we will use the task-boot image which will have all the required command

Reply:
It might be a while until we implement it in taskboot, as we haven't settled on a decision about this yet. We might also keep task-boot only about building/deploying.

Anyway, let's file an issue so we don't forget to make this avoid the git clone + dependency installation at runtime.